samples/matter/common/src/app/task_executor.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "task_executor.h"
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+constexpr size_t kTaskQueueSize = CONFIG_NCS_SAMPLE_MATTER_APP_TASK_QUEUE_SIZE;
+
+K_MSGQ_DEFINE(sTaskQueue, sizeof(Nordic::Matter::Task), kTaskQueueSize, alignof(Nordic::Matter::Task));
+
+namespace Nordic
```

Comment:
Note that starting from C++17, you could use nested namespaces:
```
namespace Nordic::Matter
{
```
It's not relevant, just fyi, that you can type fewer characters to achieve the same :).